src/state-client.ts

```diff
@@ -284,157 +323,197 @@ class StateClient {
    * local cached version of the state to the response.
    * @ignore
    */
-  public getState = (identifier: string, substate?: ServerState): Promise<any> =>
-    this.signedRequest(identifier, 'GET', substate || ServerState.ALL);
+  public getState = <ResponseType = unknown>(identifier: string, substate?: ServerState): Promise<ResponseType> =>
+    this.signedRequest<ServerState, ResponseType>(identifier, 'GET', substate || ServerState.ALL);
```

Comment:
`ServerState` shouldn't be the data type to `signedRequest` on this line. Do `this.signedRequest<undefined, ResponseType>(identifier, 'GET', substate || ServerState.ALL);`